### PR TITLE
Feature/android getting started live data

### PR DESCRIPTION
**Pull request by timMikkelsen**

### CODE REVIEW

**Review by Daniel-Nielsen — APPROVED** [review]

File: src/android/v3/getting-started/directions.md [hunk 1]
```diff
@@ -184,7 +188,7 @@ override fun onRouteResult(@Nullable route: Route?, @Nullable miError: MIError?)
 </mi-tab-panel>
 </mi-tabs>
 
-See the full implementation of theese methods here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L240-L288) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L240-L288)
+See the full implementation of theese methods here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L218-L267) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L185-L225)
```
Comment (marksdk) [inline src/android/v3/getting-started/directions.md:191]:
```suggestion
See the full implementation of these methods here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L218-L267) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L185-L225)
```

File: src/android/v3/getting-started/livedata.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  key: android-v3-getting-started-enable-live-data
+  parent: android-v3-getting-started
+  order: 170
+---
+
+{% include "src/shared/live-data/live-data-intro.md" %}
+
+{% include "src/shared/getting-started/live-data/live-position-demo-preconditions.md" %}
+
+Enabling of Live Data through `MapControl` can be done as simple as calling `mapControl.enableLiveData()` with a [Domain Type](https://app.mapsindoors.com/mapsindoors/reference/android/v3/index.html).
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:14]:
```suggestion
Enabling Live Data through `MapControl` is as simple as calling `mapControl.enableLiveData()` with a [Domain Type](https://app.mapsindoors.com/mapsindoors/reference/android/v3/index.html).
```

Is this still correct, @timMikkelsen?

File: src/android/v3/getting-started/livedata.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+We will create a new method on our `MapsActivity` called `enableLiveData()` to enable live data for the solution.
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:16]:
```suggestion
We will create a new method on our `MapsActivity` called `enableLiveData()` to enable Live Data for the Solution.
```

File: src/android/v3/getting-started/livedata.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+<mi-tabs>
+<mi-tab label="Java" tab-for="java"></mi-tab>
+<mi-tab label="Kotlin" tab-for="kotlin"></mi-tab>
+<mi-tab-panel id="java">
+<a href="https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L270-L278">MapsActivity.java</a>
+
+```java
+void enableLiveData() {
+    //Enabling live data for the three known live data domains that are enabled for this solution.
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:26]:
```suggestion
    //Enabling Live Data for the three known Live Data Domains enabled for this Solution.
```

File: src/android/v3/getting-started/livedata.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+}
+```
+
+</mi-tab-panel>
+<mi-tab-panel id="kotlin">
+<a href="https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L227-L235">MapsActivity.kt</a>
+
+```kotlin
+private fun enableLiveData() {
+    //Enabling live data for the three known live data domains that are enabled for this solution.
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:39]:
```suggestion
    //Enabling Live Data for the three known Live Data Domains enabled for this Solution.
```

File: src/android/v3/getting-started/livedata.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    mMapControl.enableLiveData(LiveDataDomainTypes.AVAILABILITY_DOMAIN)
+    mMapControl.enableLiveData(LiveDataDomainTypes.OCCUPANCY_DOMAIN)
+    mMapControl.enableLiveData(LiveDataDomainTypes.POSITION_DOMAIN)
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+By consequence, `MapControl` will manage the Live Data subscriptions needed for the current visible map and provide a default rendering of the Live Updates depending on the Domain Type.
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:49]:
```suggestion
By consequence, `MapControl` will manage the Live Data subscriptions needed for the currently visible map and provide a default rendering of the Live Data updates depending on the Domain Type.
```

File: src/android/v3/getting-started/livedata.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+In the context of your view controller showing a map, just add the call after creating your `MapControl` object used in the `Activity` in the `initMapControl()` method created earlier.
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:51]:
```suggestion
In the context of your view controller showing a map, add the call after creating your `MapControl` object used in the `Activity` in the `initMapControl()` method created earlier.
```

File: src/android/v3/getting-started/livedata.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+<mi-tab label="Java" tab-for="java"></mi-tab>
+<mi-tab label="Kotlin" tab-for="kotlin"></mi-tab>
+<mi-tab-panel id="java">
+<a href="https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L148-L168">MapsActivity.java</a>
+
+```java
+void initMapControl(View view) {
+        //Creates a new instance of MapControl
+        mMapControl = new MapControl(this);
+        //Enable live data on the map
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:63]:
```suggestion
        //Enable Live Data on the map
```

File: src/android/v3/getting-started/livedata.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+</mi-tab-panel>
+<mi-tab-panel id="kotlin">
+<a href="https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L116-L134">MapsActivity.kt</a>
+
+```kotlin
+private fun initMapControl(view: View) {
+    //Creates a new instance of MapControl
+    mMapControl = MapControl(this)
+    //Enable live data on the map
```
Comment (marksdk) [inline src/android/v3/getting-started/livedata.md:77]:
```suggestion
    //Enable Live Data on the map
```

File: src/android/v3/getting-started/search.md [hunk 10]
```diff
@@ -67,13 +68,13 @@ private fun search(searchQuery: String) {
...
+We then add an `EditorActionListener` and a `OnClickListener` to our text input field and our search button in the `onCreate` of `MapsActivity`. To call our search method with the text in the search input field. Find the full `onCreate` example here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L53-L117) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-started-android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L42-L106)
```
Comment (marksdk) [inline src/android/v3/getting-started/search.md:71]:
```suggestion
To call our search method with the text in the search input field, we then add an `EditorActionListener` and a `OnClickListener` to the text input field and the search button in the `onCreate` of `MapsActivity`. Find the full `onCreate` example here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L53-L117) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-started-android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L42-L106)
```

**Review by marksdk — CHANGES_REQUESTED** [review]

**Review by fhansenMP — APPROVED** [review]

File: src/shared/getting-started/prerequisites/api-key-mapsindoors.md [hunk 11]
```diff
@@ -1,3 +1,3 @@
 ## Get your MapsIndoors API key
 
-If you are not a customer you can use this demo MapsIndoors API key `{sdk.tutorialAPIKey}` to follow this guide or you can [contact MapsPeople](https://resources.mapspeople.com/contact-us) to get your building drawings processed and hosted by us to receive a unique API key.
+If you are not a customer you can use this demo MapsIndoors API key `{{sdk.tutorialAPIKey}}` to follow this guide or you can [contact MapsPeople](https://resources.mapspeople.com/contact-us) to get your building drawings processed and hosted by us to receive a unique API key.
```
Comment (marksdk) [inline src/shared/getting-started/prerequisites/api-key-mapsindoors.md:3]:
```suggestion
If you are not a customer, you can use this demo MapsIndoors API key `{{sdk.tutorialAPIKey}}` to follow this guide, or you can [contact MapsPeople](https://resources.mapspeople.com/contact-us) to get your building drawings processed and hosted by us to receive a unique API key.
```

**Review by marksdk — APPROVED** [review]

**Comment by marksdk** [comment]
Nice work @timMikkelsen 👏

**Pull request merged**